backend/workflow_manager/endpoint/database_utils.py

```diff
@@ -338,17 +410,59 @@ def create_table_if_not_exists(
         sql = DBConnectorQueryHelper.create_table_query(
             conn_cls=db_class, table=table_name, database_entry=database_entry
         )
-        logger.debug(f"creating table with: {sql} query")
+        logger.debug(f"creating table {table_name} with: {sql} query")
         try:
             if hasattr(engine, "cursor"):
                 with engine.cursor() as cursor:
                     cursor.execute(sql)
                 engine.commit()
             else:
                 engine.query(sql)
-        except Exception as e:
-            logger.error(f"Error while creating table: {str(e)}")
-            raise e
+        except PsycopgError.InvalidSchemaName as e:
```

Comment:
NIT: Can't we refractor this exception handling duplication?

Reply:
@kirtimanmishrazipstack Same here as above. Added how we can improve this.

Reply:
In [backend/init.py]( https://github.com/Zipstack/unstract/blob/main/backend/backend/__init__.py)  we are importing celery

- Since backend is using celery, [here](https://github.com/Zipstack/unstract/pull/330/files#diff-be08282375f716f88d5b5dfa1fda6256970f619509a9347c439736ea012dbbdc) (  backend/workflow_manager/endpoint/tests/__init__.py ) I have added init.py to add celery

- Also added empty init.py [here](https://github.com/Zipstack/unstract/pull/330/files#diff-798949c8c03722563003ba9d871ea3a726609c3a583f9d56952e8e5bd50a99a0) ( backend/workflow_manager/endpoint/tests/test_database_utils/__init__.py ) to test all classes under test_database_utils .

Reply:
@ath

Reply:
@athul-rs

Reply:
refactoring and moving to base unstract_db class is not helpful as mentioned here https://github.com/Zipstack/unstract/pull/330#discussion_r1617174048